crates/turbopack-dev-server/src/source/router.rs

```diff
@@ -125,34 +122,54 @@ impl ContentSource for RouterContentSource {
     }
 }
 
+#[turbo_tasks::value]
+struct PrefixedRouterContentSourceMapper {
+    prefix: StringVc,
```

Comment:
Seems unused? Unless you meant mean to pass the `prefix` `StringVc` to `PrefixedRouterGetContentSourceContent` instead of the `PrefixedRouterContentSourceMapperVc`?